modules/dkimPolicy.jsm

```diff
@@ -409,6 +409,162 @@ var Policy = {
+			const Ldh_str_p = `(?:[A-Za-z0-9-]*${Let_dig_p})`;
+			// "Keyword" as specified in Section 4.1.2 of RFC 5321 [SMTP].
+			const Keyword_p = Ldh_str_p;
+			// sub-domain as specified in Section 4.1.2 of RFC 5321 [SMTP].
+			const sub_domain_p = `(?:${Let_dig_p}${Ldh_str_p}?)`;
+			// obs-FWS as specified in Section 4.2 of RFC 5322
+			const obs_FWS_p = `(?:${WSP_p}+(?:\r\n${WSP_p}+)*)`;
+			// quoted-pair as specified in Section 3.2.1 of RFC 5322
+			// Note: obs-qp is not included, so this pattern matches less then specified!
+			const quoted_pair_p = `(?:\\\\(?:${VCHAR_p}|${WSP_p}))`;
+			// FWS as specified in Section 3.2.2 of RFC 5322
+			const FWS_p = `(?:(?:(?:${WSP_p}*\r\n)?${WSP_p}+)|${obs_FWS_p})`;
+			const FWS_op = `${FWS_p}?`;
+			// ctext as specified in Section 3.2.2 of RFC 5322
+			const ctext_p = "[!-'*-[\\]-~]";
+			// ccontent as specified in Section 3.2.2 of RFC 5322
+			// Note: comment is not included, so this pattern matches less then specified!
+			const ccontent_p = `(?:${ctext_p}|${quoted_pair_p})`;
+			// comment as specified in Section 3.2.2 of RFC 5322
+			const comment_p = `\\((?:${FWS_op}${ccontent_p})*${FWS_op}\\)`;
+			// CFWS as specified in Section 3.2.2 of RFC 5322 [MAIL]
+			const CFWS_p = `(?:(?:(?:${FWS_op}${comment_p})+${FWS_op})|${FWS_p})`;
+			const CFWS_op = `${CFWS_p}?`;
+			// atext as specified in Section 3.2.3 of RFC 5322
+			const atext_p = "[!#-'*-+/-9=?A-Z^-~-]";
+			// dot-atom-text as specified in Section 3.2.3 of RFC 5322
+			const dot_atom_text_p = `(?:${atext_p}+(?:\\.${atext_p}+)*)`;
+			// dot-atom as specified in Section 3.2.3 of RFC 5322
+			// dot-atom        =   [CFWS] dot-atom-text [CFWS]
+			const dot_atom_p = `(?:${CFWS_op}${dot_atom_text_p}${CFWS_op})`;
+			// qtext as specified in Section 3.2.4 of RFC 5322
+			// Note: obs-qtext is not included, so this pattern matches less then specified!
+			const qtext_p = "[!#-[\\]-~]";
+			// qcontent as specified in Section 3.2.4 of RFC 5322
+			const qcontent_p = `(?:${qtext_p}|${quoted_pair_p})`;
+			// quoted-string as specified in Section 3.2.4 of RFC 5322
+			const quoted_string_p = `(?:${CFWS_op}"(?:${FWS_op}${qcontent_p})*${FWS_op}"${CFWS_op})`;
+			const quoted_string_cp = `(?:${CFWS_op}"((?:${FWS_op}${qcontent_p})*)${FWS_op}"${CFWS_op})`;
+			// local-part as specified in Section 3.4.1 of RFC 5322
+			// Note: obs-local-part is not included, so this pattern matches less then specified!
+			const local_part_p = `(?:${dot_atom_p}|${quoted_string_p})`;
+			// token as specified in Section 5.1 of RFC 2045.
+			const token_p = "[^ \\x00-\\x1F\\x7F()<>@,;:\\\\\"/[\\]?=]+";
+			// "value" as specified in Section 5.1 of RFC 2045.
+			const value_cp = `(?:(${token_p})|${quoted_string_cp})`;
+			// domain-name as specified in Section 3.5 of RFC 6376 [DKIM].
+			const domain_name_p = `(?:${sub_domain_p}(?:\\.${sub_domain_p})+)`;
+			
+			const replyToAddrMatcher = `reply-to:${WSP_p}(${local_part_p}@${domain_name_p})\r\n`
```

Comment:
I'm ok with leaving as is, but note that if I see it correctly this will not match all valid reply-to headers.
E.g. the following is valid: `Reply-To: "noreply@mail.paypal.de" <noreply@mail.paypal.de>`. See also the notification mails from GitHub that have something more complex than this would match.

Using Thunderbird's internal parser should avoid this Problem.

Reply:
Changed to Thunderbird's internal parser :)